script.js

```diff
@@ -0,0 +1,17 @@
+const burger = document.querySelector('.burger');
+const menuMobile = document.querySelector('.menu-mobile');
+const closeIcon = document.querySelector('.close-icon');
+
+burger.addEventListener('click', () => {
+  menuMobile.classList.add('active');
+});
+
+closeIcon.addEventListener('click', () => {
+  menuMobile.classList.remove('active');
+});
+
+Window.addEventListener('resize', () => {
+  if (window.innerWidth > 768) {
+    menuMobile.classList.remove('active');
+  }
+});
```

Comment:
- [ ] Ouch! this is close, as specified in the project requirements when the user clicks any of the mobile menu options, the mobile menu should disappear. So please make sure you add that functionality to your JavaScript here. ✔
- Here's how it works currently:
  ![firefox_YMqC2fyLFh](https://user-images.githubusercontent.com/104968236/233145114-e4e26937-29e2-4474-a3d3-31644577799d.gif)